content/en/docs/reference/node/seccomp.md

```diff
@@ -0,0 +1,132 @@
+: The workload runs without any seccomp restrictions.
+
+`RuntimeDefault`
+: A default seccomp profile defined by the
+{{< glossary_tooltip text="container runtime" term_id="container-runtime" >}}
+is applied. The default profiles aim to provide a strong set of security
+defaults while preserving the functionality of the workload. It is possible that
+the default profiles differ between container runtimes and their release
+versions, for example when comparing those from
+{{< glossary_tooltip text="CRI-O" term_id="cri-o" >}} and
+{{< glossary_tooltip text="containerd" term_id="containerd" >}}.
+
+`Localhost`
+: The `localhostProfile` will be applied, which has to be available on the node
+disk (on Linux it's `/var/lib/kubelet/seccomp`). The availability of the seccomp
+profile is verified by the
+{{< glossary_tooltip text="container runtime" term_id="container-runtime" >}}
+on container creation. If the profile does not exist, then the container
+creation will fail with a `CreateContainerError`.
+
+### `Localhost` profiles
+
+Seccomp profiles are JSON files following the scheme defined by the
+[OCI runtime specification](https://github.com/opencontainers/runtime-spec/blob/f329913/config-linux.md#seccomp).
+A profile basically defines actions based on matched syscalls, but also allows
+to pass specific values as arguments to syscalls. For example:
+
+{{% code_sample file="pods/security/seccomp/profile.json" %}}
+
+The `defaultAction` in the profile above is defined as `SCMP_ACT_ERRNO` and
+should return on every not matching syscall of `syscalls.names`. The error is
+defined as code `38` via the `defaultErrnoRet` field.
+
+The following actions are generally possible:
+
+`SCMP_ACT_ERRNO`
+: Return the specified error code.
+
+`SCMP_ACT_ALLOW`
+: Allow the syscall to be executed.
+
+`SCMP_ACT_KILL_PROCESS`
+: Kill the process.
+
+`SCMP_ACT_KILL_THREAD` and `SCMP_ACT_KILL`
+: Kill only the thread.
+
+`SCMP_ACT_TRAP`
+: Throw a `SIGSYS` signal.
+
+`SCMP_ACT_NOTIFY` and `SECCOMP_RET_USER_NOTIF`.
+: Notify the user space.
+
+`SCMP_ACT_TRACE`
+: Notify a tracing process with the specified value.
+
+`SCMP_ACT_LOG`
+: Allow the syscall to be executed after the action has been logged to syslog or
+auditd.
+
+Some actions like `SCMP_ACT_NOTIFY` or `SECCOMP_RET_USER_NOTIF` may be not
+supported depending on the container runtime, OCI runtime or Linux kernel
+version being used. There may be also further limitations, for example that
+`SCMP_ACT_NOTIFY` cannot be used as `defaultAction` or for certain syscalls like
+`write`. All those limitations are defined by either the OCI runtime
+([runc](https://github.com/opencontainers/runc),
+[crun](https://github.com/containers/crun)) or
+[libseccomp](https://github.com/seccomp/libseccomp).
+
+The `syscalls` JSON array contains a list of objects referencing syscalls by
+their respective `names`. In the above example the list of syscalls to be
+allowed is using the action `SCMP_ACT_ALLOW`. It would also be possible to
```

Comment:
> In the above example the list of syscalls to be
allowed is using the action `SCMP_ACT_ALLOW`.

This is difficult to parse ...

Reply:
Changed that part :+1: